components/ryr-instructions.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+
+const InstTitleStyle = { textAlign: 'center', marginBottom: '0.8em', fontSize: '1.8em' };
+
+const InstTypes = { textAlign: 'center', marginBottom: '1.5em' };
+const InstTypes2 = { textAlign: 'center', marginBottom: '1.5em', textIndent: '-4.9em' };
+
```

Comment:
The name of the variables are not very clear. Don't use abbreviations just to save characters.

Reply:
Okay. I changed the variables' names

Reply:
Okay. I changed the variables' names